src/99-appendices/08-coordinate-systems.md

```diff
@@ -47,53 +47,82 @@ different conventions used in specific software packages developed by different
 
 Generally, across the MEG, EEG, and iEEG modalities, the first two pieces of
 information for a coordinate system (origin and orientation) are specified in
-`<datatype>CoordinateSystem`.
+`<CoordSysType>CoordinateSystem`.
 The third piece of information for a coordinate system (units) are specified in
-`<datatype>CoordinateUnits`.
-Here, `<datatype>` can be one of `MEG`, `EEG`, or `iEEG`, depending on the
-modality that is being used.
-
-Allowed values for the `<datatype>CoordinateSystem` field come from a list of
+`<CoordSysType>CoordinateUnits`.
+Here, `<CoordSysType>` can be one of of the following,
+depending on the data that is supposed to be documented:
+
+-   `MEG`
+-   `EEG`
+-   `iEEG`
+-   `Fiducials`
+-   `AnatomicalLandmark`
+-   `HeadCoil`
+-   `DigitizedHeadPoints`
+
+Allowed values for the `<CoordSysType>CoordinateSystem` field come from a list of
 restricted keywords, as listed in the sections below.
+
+Note that `Fiducials`, `AnatomicalLandmark`, `HeadCoil`, and `DigitizedHeadPoints`
+`CoordSysTypes` share the restricted keywords with the data modality they are shared with.
```

Comment:
This probably doesn't matter, but just in case it _could_ be a source of confusion...

```suggestion
`CoordSysType`s share the restricted keywords with the data modality they are shared with.
```